src/map.ts

```diff
@@ -0,0 +1,88 @@
+import 'leaflet/dist/leaflet.css';
+import L from "leaflet";
```

Comment:
This may become all of the parts of leaflet you use if you move away from the CDN.
e.g.
```ts
import {Map} from 'leaflet';

const map = new Map('map'); 
```
Note that you are then instantiating classes with their constructors, so both uppercase and new keywords are required.

Here's also where you'd import leaflet's CSS